widgetssdk/src/main/java/com/glia/widgets/chat/ChatView.kt

```diff
@@ -508,6 +508,11 @@ class ChatView(context: Context, attrs: AttributeSet?, defStyleAttr: Int, defSty
         } else {
             binding.appBarView.hideEndScreenSharingButton()
         }
+        if (chatState.isSecureMessaging) {
+            showToolbar(resources.getString(R.string.glia_messaging_title))
+        } else {
+            showToolbar(theme.appBarTitle)
+        }
```

Comment:
Let me clarify, an integrator can customize the chat screen title but cannot customize messaging screen title, right?

Reply:
For now, yes. Actually, I don't know, if should we provide the ability to configure the Messaging title.